### PR TITLE
Fabo/ update dependencies

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/connectors/tendermint-ws.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
+'use strict'
+
+/*
+* Extracted from https://github.com/nomic-io/js-tendermint
```
Comment (jbibla) [inline app/src/renderer/connectors/tendermint-ws.js]: awesome!

File: app/src/renderer/scripts/boot.js [hunk 2]
```diff
@@ -51,7 +51,12 @@ export const startApp = async (
   enableGoogleAnalytics = _enableGoogleAnalytics
 ) => {
   Vue.use(Router)
-  Vue.use(Tooltip, { delay: 1 })
+  Vue.use(Tooltip, {
+    delay: 500,
+    placement: `bottom`,
```
Comment (jbibla) [inline app/src/renderer/scripts/boot.js]: i think if you don't set this then it uses some kind of "auto" placement feature... so why always bottom?

Comment (faboweb) [reply app/src/renderer/scripts/boot.js]: I will change it back to auto. There were some bugs when updating this library. I need to revert the update. Might be interesting to migrate to another more maintained lib.

Comment (jbibla) [reply app/src/renderer/scripts/boot.js]: feel free to merge after changing this back to auto

**Pull request merged**